src/seth/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fix `--use` bug
 - `seth bundle-source` writes the contents of standard-json to the current directory to enable better sourcemaps for multi-file etherscan source code.
+- `seth estimate` now properly calculates the gas limit for transactions which may require >4m gas
```

Comment:
```suggestion
- `seth estimate` no longer sets a gas limit when estimating gas
```

I'm actually not sure where the 4 million was coming from (and couldn't reproduce locally). I suspect you had `ETH_GAS` set (either in your environment or in some `.dapprc` file)?

Reply:
Fairly sure I didn't have the `ETH_GAS` env var set! Sorry a bit low on time to try to repro it again.

In any case, the PR should be required _in case_ somebody wants to estimate gas and they have `ETH_GAS` set, right?

Reply:
Yea the 4 million is curious, but the PR is good regardless ☺️